toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -300,20 +300,208 @@ pcre-8.42-4.cm1.x86_64.rpm
+perl-Module-Loaded-0.08-463.cm1.noarch.rpm
+perl-Module-Metadata-1.000037-463.cm1.noarch.rpm
+perl-mro-1.23-463.cm1.x86_64.rpm
+perl-NDBM_File-1.15-463.cm1.x86_64.rpm
+perl-Net-1.02-463.cm1.noarch.rpm
+perl-Net-Ping-2.72-463.cm1.noarch.rpm
+perl-NEXT-0.67-463.cm1.noarch.rpm
+perl-Object-Accessor-0.48-7.cm1.noarch.rpm
+perl-ODBM_File-1.16-463.cm1.x86_64.rpm
+perl-Opcode-1.47-463.cm1.x86_64.rpm
+perl-open-1.12-463.cm1.noarch.rpm
+perl-overload-1.31-463.cm1.noarch.rpm
+perl-overloading-0.02-463.cm1.noarch.rpm
+perl-Params-Check-0.38-463.cm1.noarch.rpm
+perl-parent-0.238-463.cm1.noarch.rpm
+perl-PathTools-3.78-463.cm1.x86_64.rpm
+perl-perlfaq-5.20200523-463.cm1.noarch.rpm
+perl-PerlIO-via-QuotedPrint-0.08-463.cm1.noarch.rpm
+perl-Perl-OSType-1.010-463.cm1.noarch.rpm
+perl-ph-5.32.0-463.cm1.x86_64.rpm
+perl-Pod-Checker-1.73-463.cm1.noarch.rpm
+perl-Pod-Escapes-1.07-463.cm1.noarch.rpm
+perl-Pod-Functions-1.13-463.cm1.noarch.rpm
+perl-Pod-Html-1.25-463.cm1.noarch.rpm
+perl-podlators-4.14-463.cm1.noarch.rpm
+perl-Pod-Perldoc-3.28.01-463.cm1.noarch.rpm
+perl-Pod-Simple-3.40-463.cm1.noarch.rpm
+perl-Pod-Usage-1.69-463.cm1.noarch.rpm
+perl-POSIX-1.94-463.cm1.x86_64.rpm
+perl-Safe-2.41-463.cm1.noarch.rpm
+perl-Scalar-List-Utils-1.55-463.cm1.x86_64.rpm
+perl-Search-Dict-1.07-463.cm1.noarch.rpm
+perl-SelectSaver-1.02-463.cm1.noarch.rpm
+perl-SelfLoader-1.26-463.cm1.noarch.rpm
+perl-sigtrap-1.09-463.cm1.noarch.rpm
+perl-Socket-2.029-463.cm1.x86_64.rpm
+perl-sort-2.04-463.cm1.noarch.rpm
+perl-Storable-3.21-463.cm1.x86_64.rpm
+perl-subs-1.03-463.cm1.noarch.rpm
+perl-Symbol-1.08-463.cm1.noarch.rpm
+perl-Sys-Hostname-1.23-463.cm1.x86_64.rpm
+perl-Sys-Syslog-0.36-463.cm1.x86_64.rpm
+perl-Term-ANSIColor-5.01-463.cm1.noarch.rpm
+perl-Term-Cap-1.17-463.cm1.noarch.rpm
+perl-Term-Complete-1.403-463.cm1.noarch.rpm
+perl-Term-ReadLine-1.17-463.cm1.noarch.rpm
+perl-Test-1.31-463.cm1.noarch.rpm
+perl-Test-Harness-3.42-463.cm1.noarch.rpm
+perl-tests-5.32.0-463.cm1.x86_64.rpm
+perl-Test-Simple-1.302175-463.cm1.noarch.rpm
+perl-Test-Warnings-0.028-4.cm1.noarch.rpm
+perl-Text-Abbrev-1.02-463.cm1.noarch.rpm
+perl-Text-Balanced-2.03-463.cm1.noarch.rpm
+perl-Text-ParseWords-3.30-463.cm1.noarch.rpm
+perl-Text-Tabs+Wrap-2013.0523-463.cm1.noarch.rpm
+perl-Text-Template-1.51-3.cm1.noarch.rpm
+perl-Thread-3.05-463.cm1.noarch.rpm
+perl-Thread-Queue-3.14-463.cm1.noarch.rpm
+perl-threads-2.25-463.cm1.x86_64.rpm
+perl-Thread-Semaphore-2.13-463.cm1.noarch.rpm
+perl-threads-shared-1.61-463.cm1.x86_64.rpm
+perl-Tie-4.6-463.cm1.noarch.rpm
+perl-Tie-File-1.06-463.cm1.noarch.rpm
+perl-Tie-Memoize-1.1-463.cm1.noarch.rpm
+perl-Tie-RefHash-1.39-463.cm1.noarch.rpm
+perl-Time-1.03-463.cm1.noarch.rpm
+perl-Time-HiRes-1.9764-463.cm1.x86_64.rpm
+perl-Time-Local-1.280-463.cm1.noarch.rpm
+perl-Time-Piece-1.3401-463.cm1.x86_64.rpm
+perl-Unicode-Collate-1.27-463.cm1.x86_64.rpm
+perl-Unicode-Normalize-1.27-463.cm1.x86_64.rpm
+perl-Unicode-UCD-0.75-463.cm1.noarch.rpm
+perl-User-pwent-1.03-463.cm1.noarch.rpm
```

Comment:
Does all of this really need to be in the toolchain?

Reply:
It does -- these packages are needed by autoconf and automake to function properly. 

These components have always been in the chroot, all installed under `perl.rpm`. Despite it being more rpms, its actually less of perl that's going into the worker chroot now.